### PR TITLE
Reduce emitted occurences noise from anonymous classes

### PR DESCRIPTION
**Pull request by Strum355**
This PR addresses the noisy output to synthetic elements arising from anonymous classes and anonymous class constructors.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -103,6 +103,7 @@ lazy val plugin = project
     javaOnlySettings,
     moduleName := "semanticdb-javac",
     javaToolchainVersion := "1.8",
+    javacOptions += "-g",
```
Comment (olafurpg) [inline build.sbt:106]: Is this required for the fix or it it only needed for debugging?

File: tests/snapshots/src/main/generated/com/airbnb/epoxy/EpoxyTouchHelper.java [hunk 2]
```diff
@@ -403,26 +403,18 @@ public ItemTouchHelper andCallbacks(final DragCallbacks<U> callbacks) {
 //                    ^^^^^^^^^^^^^^^ definition local18 unresolved_type itemTouchHelper
           new ItemTouchHelper(new EpoxyModelTouchCallback<U>(controller, targetModelClass) {
```
Comment (olafurpg) [inline tests/snapshots/src/main/generated/com/airbnb/epoxy/EpoxyTouchHelper.java:404]: Nice!

**Review by olafurpg — APPROVED** [review]
Awesome, really good to see this fixed 👍

File: build.sbt (hunk 1 above)
Comment (Strum355) [reply build.sbt:106]: Not required, but makes intellij not show a constant warning in the variables view while debugging. 
Unless you've a reason we shouldn't include it, I dont see it as anything but benign

Comment (olafurpg) [reply build.sbt:106]: Sounds good, was just curious if this flag affected semanticdb

**Pull request merged**